### ISSUE TITLE
Update documentation for Console32/windows support

### ISSUE THREAD

**Issue by galri** [realm/realm-dotnet#1391]
## Goals

Make it easier to develop realm with console 32/windows. Another issue is why A console 32/windows prosject should need the package Realm.DataBinding (from nuget version 1.2).

## Expected Results
Walkthrough, requirments and list of supported windows os.
### Requirements
Microsoft Visual c++ 2015 Redistributable 14 x86 (i needed this on windows server 2008)
minimum .Net framework version ?

### Os support, windows server included
Supported:
- Supported os
- supported os

Not 
- not supported os
- another not supported os.

Could also just say from server x and windows x i guess.

### Build steps
(this is what i had to do, maybe some one else knows a better way)

1. Create Console 32 project, framework 4.5.2.
2. Reference nuget package "Realm.Database" version 1.1.1
3. set build platform to x86 or x64
4. build and run.

This worked on windows 10, but when i tried it on windows server 2008 and windows server 2012 if failed. 
![bilde](https://cloud.githubusercontent.com/assets/8320121/25888580/0fd9dd40-3567-11e7-8266-d7150d1266c6.png)

Only after installing Microsoft Visual c++ 2015 Redistributable 14 it worked.

## Actual Results

Some places it says its supported in windows
https://github.com/realm/realm-dotnet/releases under version 1.0.3

some place not:
https://github.com/realm/realm-dotnet/blob/master/README.md (...Currently, we only support Xamarin..)

No Walkthrough that i could find. Perhaps the best soution would be to add console 32/windows to https://realm.io/docs/

**Comment by fealebenpae** [comment]
Hey @galri,

We definitely should improve our Windows documentation - I guess we have been holding off because our Windows support is still incomplete: we don't yet have Sync or UWP.

Realm is a .NET Standard 1.4 library and according to https://docs.microsoft.com/en-us/dotnet/articles/standard/library the earliest version of .NET Framework that implements .NET Standard 1.4 is 4.6.1.

Re. the Visual C++ Redistributable: we can improve this if we link the C and C++ runtime libraries statically when building `realm-wrappers.dll` for windows which should remove the need to have the Redistributable package installed. We'll do this in the near future.

Hope this helps.

**Comment by galri** [comment]
It does, should have thought of checking netstandard version sooner. then i can check os
https://msdn.microsoft.com/nb-no/library/8z6watww(v=vs.110).aspx

**Comment by nirinchev** [comment]
I feel this should be fixed now - the docs state which windows versions are supported and we're statically linking Visual C++ Redistributable now.
